modules/ROOT/pages/_partials/endpoint-as2-receive.adoc

```diff
@@ -0,0 +1,78 @@
+== AS2 Receive Settings
+
+[%header,cols="3s,7a"]
+|===
+|Setting |Description
+
+| URL
+| The HTTP/S address at which you are accepting incoming AS2 messages. +
+
+Required?::
+Yes
+
+Default::
+Not applicable
```

Comment:
Can we say "None" instead of "not applicable"?

Reply:
If none communicates the fact a default is not applicable for that setting then that is fine.